meson.build

```diff
@@ -98,12 +85,6 @@ conf.set10('HAVE_SPLIT_BIN', split_bin,
            description : 'bin and sbin directories are separate')
 
 rootprefixdir = get_option('rootprefix')
```

Comment:
We can probably drop `rootprefixdir` and friends and only rely on `prefixdir, bindir, libdir`

Reply:
done